Tests/Services/LanguageProviderTest.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopware\WebInstaller\Tests\Services;
+
+use PHPUnit\Framework\Attributes\CoversClass;
+use PHPUnit\Framework\TestCase;
+use Shopware\WebInstaller\Services\LanguageProvider;
+use Symfony\Component\DependencyInjection\ParameterBag\ParameterBagInterface;
+
+#[CoversClass(LanguageProvider::class)]
+class LanguageProviderTest extends TestCase
+{
+    public function testGetSupportedLanguages(): void
+    {
+        $params = $this->createMock(ParameterBagInterface::class);
+        $params->method('get')
+            ->with('shopware.installer.supportedLanguages')
+            ->willReturn([
+                'de'    => 'de',
+                'en'    => 'en',
```

Comment:
There's one inconsistency. Here you use 'en' while the yaml says 'en-GB'. I think the yamls should be correct, so we do not have en-GB anymore, since in this will be the default after merging the language layer

Reply:
My point was that both were different.

Also, in the core, we'll use just "en" as british english, since we need to declare a base language (otherwise language layer wouldn't make any sense, when we would have no base). English is somewhat of a special case because unlike german, there's no country which speaks "general english", so there needs to be a declaration.

Therefore, `en` should be the "truth" here. But we can talk about that in the meeting tomorrow with Dominik if you want